pkg/controllers/management/clusterprovisioner/driver.go

```diff
@@ -62,38 +46,10 @@ func (p *Provisioner) getKontainerDriver(spec apimgmtv3.ClusterSpec) (*apimgmtv3
 func (p *Provisioner) driverUpdate(
```

Comment:
nit: need to update comment here as it's not relevant anymore

Reply:
done